pkgs/development/ocaml-modules/get-activity/lib.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  buildDunePackage,
+  fetchFromGitHub,
+  ppx_expect,
+  astring,
+  curly,
+  fmt,
+  logs,
+  ppx_yojson_conv,
+  yojson,
+  alcotest
+}:
+
+buildDunePackage rec {
+  pname = "get-activity-lib";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "tarides";
+    repo = "get-activity";
+    rev = version;
+    hash = "sha256-QU/LPIxcem5nFvSxcNApOuBu6UHqLHIXVSOJ2UT0eKA=";
+  };
+
+  minimalOCamlVersion = "4.08";
+
+  buildInputs = [ ppx_expect alcotest ];
+
+  propagatedBuildInputs = [
+    astring
+    curly
+    fmt
+    logs
+    ppx_yojson_conv
```

Comment:
Propagating `ppx_yojson_conv_lib` (instead of `ppx_yojson_conv`) is enough.

Reply:
How did you figure this out?

Reply:
Reading the `META` file:

> cat /nix/store/…-ocaml5.1.1-get-activity-lib-2.0.1/lib/ocaml/5.1.1/site-lib/get-activity-lib/META
description = ""
requires = "astring curly fmt logs ppx_yojson_conv_lib yojson"